shell/platform/embedder/vsync_waiter_embedder.cc

```diff
@@ -42,7 +42,8 @@ bool VsyncWaiterEmbedder::OnEmbedderVsync(
         auto vsync_waiter = weak_waiter->lock();
         delete weak_waiter;
         if (vsync_waiter) {
-          vsync_waiter->FireCallback(frame_start_time, frame_target_time);
+          vsync_waiter->FireCallback(frame_start_time, frame_target_time,
```

Comment:
Is there ever a case where we aren't passing frame_target_time twice?

Reply:
In Android, I am setting up 2-frame-depth-pipeline now. So the next_frame_start_time is before frame_target_time.